bot_buycode.cpp

```diff
@@ -110,15 +110,18 @@ bool GoodWeaponForTeam(int weapon, int team) {
                 case CS_WEAPON_AK47:
                     return false;
                     break;
-                case CS_WEAPON_ELITE:
+                case CS_WEAPON_DEAGLE:

```

Comment:
are you sure about these changes in `GoodWeaponForTeam` ?

Reply:
Well, those Dual Elite Berettas aren't as powerful as the Desert Eagle. Also, I had older bots like POD-Bot that can allow rapid firing those Elites. Plus those bots should really save some money for MP5s as their minimum primary weapon as Elites costs $800 compared to the Deagles for $650, which saves them $150 for like extra ammo for their MP5s etc. Even the FN Five-seveN pistols are weak and are extortionate. 

Besides we should allow your bots to avoid buying lame weapons as well as not use their pistols too often.